### ISSUE TITLE
program4

### ISSUE THREAD

**Issue by muhafiz11** [muhafiz11/all10programs#5]
from sklearn.preprocessing import StandardScaler, MinMaxScaler

x=[[1, 400], [2, 800], [2, 1000], [3, 1200], [1, 550], [3, 1500]]

scalar = StandardScaler()
x_standardized = scalar.fit_transform(x)
print("Standardized Data:\n",x_standardized)

normalizer = MinMaxScaler()
x_normalized = normalizer.fit_transform(x)
print("Normalized Data:\n", x_normalized)
